Deployment/Kubernetes/install-kubeadm.sh

```diff
@@ -31,11 +31,10 @@ then
 	sudo apt-get install -y apt-transport-https ca-certificates curl
 
 	# Download the Google Cloud public signing key:
-	sudo curl -fsSLo /usr/share/keyrings/kubernetes-archive-keyring.gpg https://packages.cloud.google.com/apt/doc/apt-key.gpg
```

Comment:
I suggest, in first stage download gpg key, and then add this key, if you want, with sudo `apt-key add`

Reply:
Why does not save this key in /usr/share/keyrings/kubernetes-archive-keyring.gpg ?